classes/cluster/mk22_full_scale/stacklight/init.yml

```diff
@@ -14,6 +14,14 @@ parameters:
     influxdb_database: lma
     influxdb_user: lma
     influxdb_password: lmapass
+    nagios_host: ${_param:stacklight_monitor_address}
+    nagios_status_port: 8001
+    nagios_username: nagiosadmin
+    nagios_password: secret
+    nagios_notification_smtp_server: 127.0.0.1
+    nagios_notification_from: 'nagios@localhost'
+    nagios_notification_email: 'root@localhost'
+    nagios_default_host_alarm_clusters: 00-clusters
```

Comment:
The above two are already defined in the nagios system class. I think we can just use the defaults that are provided there, no?